src/qt/intro.cpp

```diff
@@ -23,7 +23,7 @@
 
 static const uint64_t GB_BYTES = 1000000000LL;
 /* Minimum free space (in GB) needed for data directory */
-static const uint64_t BLOCK_CHAIN_SIZE = 200;
+constexpr uint64_t BLOCK_CHAIN_SIZE = 220;
```

Comment:
`static`?

Reply:
Is it possible for a constexpr to be non-static?